### DISCUSSION TITLE
Please help me align Structural Frame Origin ?

### DISCUSSION THREAD

**Discussion by limkaflyer** [JSBSim-Team/jsbsim#875] [Q&A] [answered]
I have pretty much tried everything but I can't seem to align the structural frame origin. As you can see the reference points are a mess and adjusting xyz axis doesnt fix this. Please assist. Thanks
![Desktop Screenshot 2023 04 01 - 20 06 48 77](https://user-images.githubusercontent.com/117994103/229298629-01b8bc35-0dc1-4b65-87a2-a90ebab8ebf9.jpg)

**Comment by gallonmate** [comment, marked as answer]
It looks like you messed up the settings of the Skeletal Mesh, which is the visual 3d model airplane. It should look like this:
![image](https://user-images.githubusercontent.com/28702753/229413390-2976c8da-a751-406e-b70f-1235ecb14156.png)

**Reply by limkaflyer** [reply]
Thanks for helping out. This is exactly what happened. The skeletal mesh got rotated on import. Fixed it now ! Thanks

**Comment by agodemar** [comment]
@limkaflyer could you please elaborate more on what a "mess" is? Is it a mess on your side, or on JSBSim side?

**Reply by limkaflyer** [reply]
Was definitely a mess on my side. Fixed now. Thanks
